clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -238,11 +237,13 @@ public String toString() {
          * incremental fetch requests (see below).
          */
         private LinkedHashMap<TopicPartition, PartitionData> next;
+        private Map<Uuid, String> topicNames;
         private final boolean copySessionPartitions;
         private int partitionsWithoutTopicIds = 0;
 
         Builder() {
             this.next = new LinkedHashMap<>();
+            this.topicNames = new HashMap<>();
```

Comment:
I added the initialization in the other builder since we were missing it.

Reply:
Thanks!